web/src/engine/main/src/keymanEngine.ts

```diff
@@ -197,7 +210,12 @@ export default class KeymanEngine<
       // Always (temporarily) hide the OSK when loading a new keyboard, to ensure
       // that a failure to load doesn't leave the current OSK displayed
       this.osk?.startHide(false);
-    })
+    });
+
+    this.keyboardRequisitioner.cache.on('keyboardAdded', (keyboard) => {
```

Comment:
Is this supposed to be camelCase unlike all the other event names?

Reply:
Might be worth a polish pass at some point; that's how it is currently specified within `engine/package-cache`'s `StubAndKeyboardCache` class.

Fortunately, whenever such a polish pass happens, all the event typing that's been put in place will rapidly raise compile errors anywhere we miss a needed change.